index.js

```diff
@@ -74,18 +122,27 @@ async function processFile(fileName,
 	}
 
 	let type = determineFileType(sourcePath);
-	if(type && plugins[type]) {
-		let plugin = plugins[type];
-		content = await plugin(content);
+	if(type && compactors[type]) {
+		let compactor = compactors[type];
+		content = await compactor(content);
 	}
 
+	/**
+	 * @type WriteFileOpts
+	 */
 	let options = { targetDir };
 	if(fingerprint !== undefined) {
 		options.fingerprint = fingerprint;
 	}
 	return assetManager.writeFile(targetPath, content, options);
 }
 
+/**
+ * The filetype is the lower case file extension
```

Comment:
```suggestion
 * The file type is the lowercase file extension
```

or even more abstract, without revealing implementation details:

```suggestion
 * The file type is derived from the respective file extension
```